ruby_review_week_four.rb

```diff
@@ -1,37 +1,145 @@
+
 
 # 8. Define a class called fish, and leave it empty.
+# class Fish
+
+# end
 
 # 9. How would you use that class to make three fish objects?
 
+# fish1 = Fish.new
+# fish2 = Fish.new
+# fish3 = Fish.new
+
 # 10. Add two attributes to your fish class, name and species, and make them accessible for reading and writing.
+class Fish
+  attr_accessor :name, :species
+  def initialize()
+    @name = name
+    @species = species
+  end
+
+end
+
+# fish1 = Fish.new("goldie", "koi")
+# puts fish1.name
+# fish1.name = "whitey"
+# puts fish1.name
 
 # 11. 
 
 # a. Create one fish object and store it in a variable, and 
 
+fish1 = Fish.new
+
 # b. on a second line, give that fish object’s name attribute the value of ‘Fred’ (assume you have access).
 
+fish1.name = "Fred"
+puts fish1.name
+
 # 12. Define a hash called ‘student’. This hash should have two keys, one called name, and the other called age, and assign a string to the name key, and an integer to the age key. 
 
+student = { name: "Mark", age: 27}
+
 # 13. Write a Ruby method that takes three arguments and returns the sum (total) of those three arguments.
 
+def add_three(a, b, c)
+  return a + b + c
+end
+
+puts add_three(1, 1, 2)
+
 # 14. Write a Ruby method that takes an array and a string as arguments, and returns true if the string is in the array, and false if it is not.
 
+def method(arr, string)
+  return_this = false
+  arr.each do |item|
+    if item == string
+      return_this = true
+    end
+  end
+  return return_this
+end
+
+arr1 = [ "mark", "john", "bill"]
+
+# puts method(arr1, "john")
+
+
 # 15. Write a method that takes one argument, a number, and that returns true if the number is divisible by three, and false otherwise. 
 
-# 16. You must use your previous method in this next question. Write another method that takes an array (of numbers) as an argument. This method will use your first method to return two arrays (within one array - an array of arrays). The first array contains the numbers that are divisible by 3, and the other contains the rest of the numbers. For example, if you pass the second method the array [9, 4, 3, 5, 30, 2, 8] it would return [[9, 3, 30],[4, 5, 2, 8]].
\ No newline at end of file
+puts "----------"
+def one_arg(num)
+  if num % 3 == 0 
+    return true
+  else 
+    return false
+  end
+end
+
+puts one_arg(9)
+puts one_arg(5)
+
+# 16. You must use your previous method in this next question. Write another method that takes an array (of numbers) as an argument. This method will use your first method to return two arrays (within one array - an array of arrays). The first array contains the numbers that are divisible by 3, and the other contains the rest of the numbers. For example, if you pass the second method the array [9, 4, 3, 5, 30, 2, 8] it would return [[9, 3, 30],[4, 5, 2, 8]].
+
+puts "-------"
+
+def next_method(arr)
+  arr_3 = []
+  arr_not_3 = []
+
+  arr.each do |number|
+    if one_arg(number) == true
```

Comment:
love this! be careful with the way in which you did the conditional `one_arg(number) == true` you could have just done this `one_arg(number)` without the `==` as it would have coerced to a boolean